_announcements/index.md

```diff
@@ -7,7 +7,7 @@ show-title: true
 ---
 <ul>

```

Comment:
Are we going to have a UI for this page that is similar to other pages, possibly with a photo? One or two explanatory sentences at the top of the page would be helpful.

Will this page have a link to an archive of past announcements?